.github/workflows/tests.yml

```
@@ -0,0 +1,39 @@
+name: Tests
+
+on:
+  push:
+  pull_request:
```

```suggestion
on: [ push, pull_request ]
```
I usually prefer a single-line list for less than 3 items.